Contributing.md

```diff
@@ -76,6 +83,36 @@ If you're reading this, you're probably creating a Pull Request or planning to d
 
 13. Create a PR to develop repository.
 
+# ERROR ❎ AND SOLUTION ✅ FOR COMMON PROBLEMS DURING SETUP 💻
+
+ 1. Please remember to include an .env file in both the frontend and backend directories. 
+ You can reference any two of the environment variables present in Discord for your configuration.
+
+ 2.Ensure Docker is running throughout the setup process. Make sure both the Redis stack and Progress 
+ components are up and running.
+
+ 3. To resolve port 3333 issues, use netstat to find the PID associated with it, then taskkill to terminate the process using that PID.
+   For Windows 
+
+     ```bash
+   "netstat -ano | findstr :3333"
+    "taskkill /PID <PID> /F"
+
+   ```
+
+   For Linux
+
+   ```bash
+   "sudo lsof -i | grep 3333"
+   "kill <PID>"
+   ```
+
+ 4. Right-click on homepage, inspect, navigate to console, access application > cookies, delete JWT files to troubleshoot login issues.
+
+
+ 5. Please use ESLint for code quality checks and Prettier for consistent code formatting before submitting pull requests.   This  ensures a clean and readable codebase. Thank you!
+   
+
```

Comment:
Let's keep this section at the last. Also, divide the section into 2 issues i.e
 1. Error while trying to login
 2. Blocked ports error

Reply:
After the changes @Pratik2026  ,You can include the command for the fifth point , as well , and @Pratik2026 do you think this would be better in a table format ?